src/routes/contacts/Graph.svelte

```diff
@@ -0,0 +1,82 @@
+<script>
+  import Icon from '$lib/component/Icon.svelte';
+  import FaBrandsInstagram from 'svelte-icons-pack/fa/FaBrandsInstagram';
+  import FaBrandsTwitter from 'svelte-icons-pack/fa/FaBrandsTwitter';
+  import FaBrandsFacebook from 'svelte-icons-pack/fa/FaBrandsFacebook';
+  import FaBrandsGithub from 'svelte-icons-pack/fa/FaBrandsGithub';
+  import IoMail from 'svelte-icons-pack/io/IoMail';
+  import FaBrandsLinkedin from 'svelte-icons-pack/fa/FaBrandsLinkedin';
+  import { copyToClipboard } from '$lib/utils';
+
+  const coords = [
+    [1, 1],
+    [6, 3],
+    [2, 5],
+    [10, 6],
+    [1, 8],
+    [7, 8]
+  ];
+  const lines = [
+    [0, 2],
+    [2, 1],
+    [1, 5],
+    [5, 3],
+    [2, 4]
+  ];
+  let innerWidth = 0;
+  let coefficient = 8;
+  const iconSize = 8;
+
+  const socials = [
+    { url: 'https://www.instagram.com/niaefeup/', icon: FaBrandsInstagram },
+    { url: 'https://github.com/NIAEFEUP', icon: FaBrandsGithub },
+    { url: 'https://www.facebook.com/NIAEFEUP', icon: FaBrandsFacebook },
+    { url: 'ni@aefeup.pt', icon: IoMail },
```

Comment:
the email should be copied to the clipboard, that doesn't seem to be working

Reply:
It's not clickable because the logo image is overlapping it (#146). Otherwise, it is working.